RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -155,3 +211,4 @@
     )
 
 phase2_hgcalV9.toModify( HGCalRecHit , thicknessCorrection = cms.vdouble(0.759,0.760,0.773) ) #120um, 200um, 300um
+phase2_hgcalV10.toModify( HGCalRecHit , thicknessCorrection = cms.vdouble(0.781,0.775,0.769) ) #120um, 200um, 300um
```

Comment:
to be safer with types, I suggest that this is changed to 
```thicknessCorrection = [0.781,0.775,0.769]``` so that the type is deduced from the original parameter definition.